SocialCareCaseViewerApi/V1/Gateways/ProcessDataGateway.cs

```diff
@@ -138,6 +140,17 @@ public ProcessDataGateway(ISccvDbContext sccvDbContext)
             return new Tuple<IEnumerable<CareCaseData>, int>(response, totalCount);
         }
 
+        public CareCaseData GetCaseById(string recordId)
+        {
+            var collection = _sccvDbContext.getCollection();
+            var filter = Builders<BsonDocument>.Filter.Eq("_id", ObjectId.Parse(recordId));
+            // var query = collection.Find(filter).FirstOrDefault();
+            var query = collection.AsQueryable().Where(db => filter.Inject());
+
+            var result = query.ToList();
+            var response = ResponseFactory.ToResponse(result);
```

Comment:
DocumentNotFoundException can be thrown here if that pattern is followed.

Reply:
Added